WinRT.Runtime/Projections.cs

```diff
@@ -206,6 +207,7 @@ private static bool IsTypeWindowsRuntimeTypeNoArray(Type type)
                 || type.GetCustomAttribute<WindowsRuntimeTypeAttribute>() is object;
         }
 
+        // Use TryGetCompatibleWindowsRuntimeTypesForVariantType instead.

```

Comment:
is the comment a TODO?

Reply:
No, the issue is this API was marked public even though we used it is only internally.  Since we don't anyone else to use it or in cross module scenarios, we can accept the breaking change and remove the old version of the API.  But before I did that, I wanted to make sure there were concerns with that, so I instead put a comment.